parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -137,11 +137,13 @@ MASTER_ADDONS_CONFIG_PLACEHOLDER
     ExecStartPre=/usr/bin/docker pull {{WrapAsVariable "kubernetesHyperkubeSpec"}}
     ExecStartPre=/usr/bin/docker run --rm -v /tmp/kubectldir:/opt/kubectldir {{WrapAsVariable "kubernetesHyperkubeSpec"}} /bin/bash -c "cp /hyperkube /opt/kubectldir/"
 {{if .MasterProfile.IsCoreOS}}
+    ExecStartPre=/bin/cp /tmp/kubectldir/hyperkube /opt/kubelet
     ExecStartPre=/bin/mv /tmp/kubectldir/hyperkube /opt/kubectl
-    ExecStart=/bin/chmod a+x /opt/kubectl
+    ExecStart=/bin/chmod a+x /opt/kubelet && /bin/chmod a+x /opt/kubectl
 {{else}}
+    ExecStartPre=/bin/cp /tmp/kubectldir/hyperkube /usr/local/bin/kubelet
```

Comment:
Let's do this in one line to make it a little clearer that we are copying the same file to two different places. One example:

`for dest in /usr/local/bin/kubelet /usr/local/bin/kubectl ; do cp /tmp/kubectldir/hyperkube $dest; done`

And then just add a cleanup line after, like:

`rm -Rf /tmp/kubectldir`

(but replace kubectldir with the new, general name in both instances :) )

Reply:
I don't really find that
 ```
ExecStartPre=for dest in /usr/local/bin/kubelet /usr/local/bin/kubectl ; do /bin/cp /tmp/hyperkube/hyperkube $dest; done
ExecStartPre=rm -Rf /tmp/kubectldir
```

is much more readable than
```
ExecStartPre=/bin/cp /tmp/hyperkubedir/hyperkube /usr/local/bin/kubelet
ExecStartPre=/bin/mv /tmp/hyperkubedir/hyperkube /usr/local/bin/kubectl
```

How about: 
```
ExecStartPre=/bin/mv /tmp/hyperkubedir/hyperkube /usr/local/bin/kubelet
ExecStartPre=/bin/cp /usr/local/bin/kubelet /usr/local/bin/kubectl
```
?

Reply:
I mean I see where you're coming from in terms of logic, I just find that it makes the syntax more complex to have a for loop for only two files. Please feel free to disagree with me :)